src/cascadia/TerminalApp/Profile.cpp

```diff
@@ -201,7 +201,12 @@ TerminalSettings Profile::CreateTerminalSettings(const std::unordered_map<std::w
 
     if (_schemeName)
     {
-        const auto found = schemes.find(_schemeName.value());
+        // TODO: I don't love that the Profile need to know that this is a map
+        // of (lowercase names)->Schemes
+        auto ourSchemeName{ _schemeName.value() };
+        std::transform(ourSchemeName.begin(), ourSchemeName.end(), ourSchemeName.begin(), std::towlower);

```

Comment:
can we use `_FindMatchingColorScheme` here?

Reply:
I'm gonna make it happen, but it's gonna involve adding an interface to globals (`IColorSchemeProvider`) with a single method `LookupColorScheme`, and then pass the globals as the param to this method, instead of passing `globals._colorschemes`

Reply:
Okay, I did this, lemme know how palatable that is, and if it's not horrifying, I'll doc it up